unblob/models.py

```diff
@@ -243,20 +243,15 @@ def parse_header(self, file: File, endian=Endian.LITTLE):
 
 
 class Handlers:
```

Comment:
At this point this could be just a plain list. We would loose the immutability aspects but I am not sure having it worth the extra complexity

Reply:
done. kept Handlers as a type but it is only a tuple definition